python/cudf/cudf/tests/test_reductions.py

```diff
@@ -175,7 +175,14 @@ def test_datetime_unsupported_reductions(op):
     try:
         getattr(psr, op)()
     except Exception as e:
-        with pytest.raises(type(e), match=re.escape(str(e))):
+        with pytest.raises(
+            type(e),
+            match=re.escape(
+                str(e).replace("median", "quantile")
```

Comment:
This feels like we're patching a test instead of patching the error message in a function?